tests/integration/test_backup_restore_on_cluster/test_disallow_concurrency.py

```diff
@@ -133,9 +133,21 @@ def test_concurrent_backups_on_same_node():
     )
     assert status in ["CREATING_BACKUP", "BACKUP_CREATED"]
 
-    error = nodes[0].query_and_get_error(
-        f"BACKUP TABLE tbl ON CLUSTER 'cluster' TO {backup_name}"
-    )
+    try:
+        error = nodes[0].query_and_get_error(
+            f"BACKUP TABLE tbl ON CLUSTER 'cluster' TO {backup_name}"
+        )
+    except Exception as e:
+        status = (
+            nodes[0]
+            .query(f"SELECT status FROM system.backups WHERE id == '{id}'")
+            .rstrip("\n")
+        )
+        # It is possible that the second backup was picked up first, and then the async backup
+        if status == "CREATING_BACKUP" or status == "BACKUP_FAILED":
```

Comment:
If the second backup succeeds and the first backup has the `CREATING_BACKUP` status then the first backup must fail soon (they have the same destination), and it's better to check that in the test too.

Reply:
If the second backup succeeds, then there are 2 posibilites:
i. first backup didn't start, and has not yet started , thus the status "CREATING_BACKUP"
ii. first backup started after second backup, then as you said it would fail , thus the status "BACKUP_FAILED"
Hope that addresses your comment

Reply:
Yes, but let's consider the "case i". The status is `CREATING_BACKUP` because of the first backup has not yet started. However what are we expecting from the first backup next at this point? Can it succeed? No. It can only fail. So what I suggest that if the status is `CREATING_BACKUP` we can make the test wait a little bit and check the status again. It must come to "BACKUP_FAILED".